doc/define/serialize.md

```diff
@@ -0,0 +1,102 @@
+@property can-define-object/define/serialize serialize
+@parent can-define-object/object.behaviors
+
+@description serialize
+
+Defines custom serialization behavior for a property.
+
+@signature `serialize( currentValue, propertyName )`
+
+  Specifies the serialized value of a property.
+
+  ```js
+  import { DefineObject } from "can";
+
+  class MyMap extends DefineObject {
+    static define = {
+      serialize( currentValue, propertyName ) {
+        console.log( currentValue ); //-> "Value"
```

Comment:
Can we show an example that actually serializes?

Reply:
This is shown below. Maybe that's enough, but it seems like this first example would be more useful if it serialized to a value also.